### ISSUE TITLE
Files getting corrupted

### ISSUE THREAD

**Issue by drunkwizaard** [ALIILAPRO/megabox#1]
Hi,
The files are getting corrupted after download is finished. I tested few links from my mega and created some demo files and one was a zip file and another was a text file, both were corrupted.

Can you please confirm if its working properly.

**Comment by sapthesh** [comment]
Hi,

I confirm the same.. 
The files are corrupted once the download is complete.

**Comment by ALIILAPRO** [comment]
Duplicate of # Files are encrypted, I'm trying to fix that

**Comment by ALIILAPRO** [comment]
Files are encrypted, I'm trying to fix that

**Comment by sapthesh** [comment]
Yes,
I totally Forgot about the Encryption, but the idea is awesome.

**Comment by ALIILAPRO** [comment]
Yeah, thanks.

**Comment by kularizky** [comment]
yes, same issue here.

trying to download "Welcome to MEGA.pdf", only 1.42 MB but it's corrupted.

I plan to download file over 20 GB, but not yet happen.

I hope you can fix it soon.
Excellent tool.
